Google.PowerShell.DotnetCore/Google.PowerShell.DotnetCore.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Google Cloud Tools for PowerShell, targeting the .NET Core runtime</Description>
+    <Copyright>Google Inc. 2016</Copyright>
+    <AssemblyTitle>Google Cloud Tools for PowerShell</AssemblyTitle>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DefineConstants>$(DefineConstants);CORECLR</DefineConstants>
+    <AssemblyName>Google.PowerShell</AssemblyName>
+    <PackageId>Google.PowerShell.DotnetCore</PackageId>
+    <RuntimeIdentifiers>win81-x64;win10-x64;ubuntu.16.04-x64;ubuntu.14.04-x64;osx.10.11-x64</RuntimeIdentifiers>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
+    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\Google.PowerShell\Common\*.cs;..\Google.PowerShell\Compute\*.cs;..\Google.PowerShell\Dns\*.cs;..\Google.PowerShell\Properties\*.cs;..\Google.PowerShell\Provider\*.cs;..\Google.PowerShell\Sql\*.cs;..\Google.PowerShell\Storage\*.cs;..\Google.PowerShell\Logging\*.cs;..\Google.PowerShell\PubSub\*.cs;..\Google.PowerShell\BigQuery\*.cs" Exclude="bin\**;obj\**;**\*.xproj;packages\**" />
```

Comment:
Cant this be reduced? I'm not familiar with this but would ..\Google.PowerShell\**\*.cs; work? and maybe exclude others?

Reply:
I tried but it doesn't work.